source/FreeRTOS_TCP_IP_IPV6.c

```diff
@@ -139,7 +137,7 @@ BaseType_t xProcessReceivedTCPPacket_IPV6( NetworkBufferDescriptor_t * pxDescrip
 
         /* Find the destination socket, and if not found: return a socket listing to
          * the destination PORT. */
-        pxSocket = ( FreeRTOS_Socket_t * ) pxTCPSocketLookup( ulLocalIP, usLocalPort, xRemoteIP, usRemotePort );
+        pxSocket = ( FreeRTOS_Socket_t * ) pxTCPSocketLookup( 0U, usLocalPort, xRemoteIP, usRemotePort );
```

Comment:
The first parameter indeed refers to the local IP-address, but it is not used.
Within FreeRTOS+TCP, you can not bind to a specific IP-address, only to a port number.

Reply:
Thanks for comment. That's why I just put a 0 here.